content/admin/policy_mgmt/Exclusions.md

```diff
@@ -103,13 +103,13 @@ Each exclusion can be enabled or disabled for either Assess or Protect mode, dep
 
 <a href="assets/images/Application-policy-exclusions-tab.png" rel="lightbox" title="Enable or disable exclusions in an application Policy tab"><img class="thumbnail" src="assets/images/Application-policy-exclusions-tab.png"/></a>
 
-Alternatively, you can see a global list of existing exclusions across all applications under **Policy Management > Application Exclusions** via the User Menu. Each exclusion can also be edited and toggled off for each mode on this page. 
+Alternatively, you can see a global list of existing exclusions across all applications under **Policy Management > Application Exclusions** via the user menu. Each exclusion can also be edited and toggled off for each mode on this page. 
 
 <a href="assets/images/Application-exclusions.png" rel="lightbox" title="Manage exclusions in Policy Management"><img class="thumbnail" src="assets/images/Application-exclusions.png"/></a>
 
 ## Regular Expression Quick Reference
 
-| | | example pattern | example match |
+| ? | ? | example pattern | example match |
 |-|-|-|
```

Comment:
@argonotix What should the headers be for the first two columns here?

Reply:
effect, pattern